stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/DropdownFieldUI.kt

```diff
@@ -116,6 +117,7 @@ fun DropDown(
                     onClickLabel = stringResource(R.string.stripe_change),
                     onClick = { expanded = true },
                 )
+                .testTag("DropDown:${if (controller.tinyMode) "tiny" else "normal"}")
```

Comment:
Why do we need the distinction?

Reply:
There's another drop down on the page for country (which isn't tiny) but the phone number one is tiny.

Reply:
Fair!